### DISCUSSION TITLE
A more powerful compact header.

### DISCUSSION THREAD

**Discussion by KTibow** [home-assistant/frontend#7480] [Dashboard]
Compact header was recently added by maykar. It doesn't apply to edit mode, though. A lot of people use edit mode a lot, and making it more compact would be nice. Also, you can't make it go to the bottom, which a lot of people like and think is easier on their fingers on mobile.

**Comment by bramkragten** [comment]
Hey Kendell,

Could you please pay attention to the tone of your requests, people might take offense from things like `we need` and `such an ugly`.

Thanks!

**Reply by KTibow** [reply]
Sorry. I'll edit that in.

**Reply by zsarnett** [reply]
Edit mode should be made a color that is different from the other colors. It's meant to distinguish the different modes

**Reply by KTibow** [reply]
There's multiple possible ways to distinguish edit mode. Maybe it's just my personal opinion.

**Reply by KTibow** [reply]
Or maybe a theme variable for `app-header-edit-mode-background-color`, but I can't select multiple categories.

**Reply by zsarnett** [reply]
Edit mode is not used to be "pretty". Its not used by a common user. Its used when editing. Hopefully you arent always in edit mode when you are showing your dashboards

**Reply by KTibow** [reply]
Thanks for the feedback. I've removed that after thinking about it a lot.

**Comment by zsarnett** [comment]
Edit mode should be made a color that is different from the other colors. It's meant to distinguish the different modes. So the "kind of ugly color" now (since you so graciously change the wording to make it SO much better) is something to be discussed but should be an "ugly" or different color to note the difference.

Edit Mode is a different mode. Not a mode that is shown all the time. I do not see the worth in changing anything about the header in Edit Mode. With the header being bigger we have more real estate to show more edit controls in the header.

The header being on the bottom for Mobile is something that we should do, IMO. It should default to the bottom for mobile. (and I don't think we should add config to change that). It isn't best practice anymore to have a hamburger menu at the top. But to use swipe gestures for the menu and bottom nav.

**Reply by KTibow** [reply]
- I've removed that now.
- Well I guess so, but in my opinion a CSS-modded compact edit header looks fine.
- Config would be nice for me, I wouldn't want to force users to re-adapt. Also, maybe the swipe gestures should be documented somewhere.
